contentcuration/contentcuration/frontend/shared/views/errors/GenericError.vue

```diff
@@ -1,37 +1,34 @@
 <template>
 
   <div>
-    <AppError>
+    <StudioAppError>
       <template #header>
         {{ $tr('genericErrorHeader') }}
       </template>
       <template #details>
         {{ $tr('genericErrorDetails') }}
       </template>
       <template #actions>
-        <VLayout
-          column
-          align-center
-        >
-          <div class="mb-3">
-            <VBtn
-              color="primary"
-              @click="reloadPage"
-            >
-              {{ $tr('refreshAction') }}
-            </VBtn>
-            <VBtn v-bind="backHomeLink">
-              {{ $tr('backToHomeAction') }}
-            </VBtn>
-          </div>
+        <KButtonGroup>
+          <KButton
+            :primary="true"
+            :text="$tr('refreshAction')"
+            @click="reloadPage"
+          />
+          <KButton
```

Comment:
To maintain the link-type behaviour here, this needs to be a KRouterLink instead of a KButton, with the `backHomeLink` object bound to the `to` prop:

```

```suggestion
          <KRouterLink
            appearance="raised-button"
            :to="backHomeLink"
            :text="$tr('backToHomeAction')"
          />
```

Alternatively, if that doesn't play nicely with `KButtonGroup` you can bind the @click event of the KButton to trigger programmatic navigation with Vue Router https://v3.router.vuejs.org/guide/essentials/navigation.html

```
@click="$router.push(backHomeLink)"
```

in place of the v-bind.